src/components/profile/edit/EditMutationWrapper.vue

```diff
@@ -131,25 +130,6 @@ export default {
     return {
       displayProfile: DISPLAY_PROFILE,
       mutateProfile: MUTATE_PROFILE,
-      ...Object.entries(this.initialValues).reduce(
```

Comment:
@Gregoor do you remember why that's there?

Reply:
initialization of those values was fairly long and repetetive, so I opted for this.